index.js

```diff
@@ -0,0 +1,21 @@
+let listOfAwesomeWomenArray = [];
+
+const addToList = () => {
+  let inputValue = document.getElementById('inputText');
+
+  if (inputValue.value === '') {
```

Comment:
This is really good! I like that you had an extra check for when the input value is falsy. Good job!